docs/CHANGELOG.rst

```diff
@@ -56,6 +56,7 @@ Added
 -----
 
 * Add Ubuntu 17.10 base Docker image
+* Add ``species`` and ``organism`` output fields to ``macs14`` process
```

Comment:
Not organism, a genome build output was added. Please change the log entry and also update the commit text.